onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -418,11 +422,16 @@ ORT_STATUS_PTR LoadAndInitializeSession(_In_ const OrtEnv* /*env*/, _In_ const O
     }
   }
 
+#if defined(ORT_MINIMAL_BUILD)
+  // TODO: Add path to load from ORT format model
```

Comment:
nit, can we always have either #if !defined(ORT_MINIMAL_BUILD) first for consistency?
It is defined(ORT_MINIMAL_BUILD) first here, but !defined(ORT_MINIMAL_BUILD) first in the code below